src/lib.rs

```diff
@@ -219,29 +220,36 @@ impl Game {
         }
     }
 
-    pub fn simulate(
-        &mut self,
-        elapsed: f64,
-        up: bool,
-        down: bool,
-        left: bool,
-        right: bool,
-        q: bool,
-    ) -> Option<String> {
-        let next_input = InputState {
+    fn local_player(&mut self) -> Option<&mut Player> {
```

Comment:
Is this better than holding a reference to the local player directly?

Reply:
Like, adding an `&Player` field on `Game`? That'd be super convenient, but I _think_ that the borrow checker wouldn't like it? Something to experiment with!